stable/contour/crds/httpproxies.yaml

```
@@ -1,14 +1,12 @@
-{{- if .Values.customResourceDefinitions.create -}}
 apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
   name: httpproxies.projectcontour.io
   labels:
-    component: httpproxy
-    app.kubernetes.io/name: contour
-    app.kubernetes.io/instance: {{ .Release.Name }}
-    app.kubernetes.io/managed-by: {{ .Release.Service }}
-    helm.sh/chart: {{ include "contour.chart" . }}
+    app: contour
```

```suggestion
    app.kubernetes.io/name: contour
```

See https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/#labels